utils/gz/crc_combine.cc

```diff
@@ -214,6 +214,9 @@ u32 fast_crc32_combine(u32 crc, u32 crc2, ssize_t len2) {
 #include <zlib.h>
 
 u32 fast_crc32_combine(u32 crc, u32 crc2, ssize_t len2) {
+    if (len2 == 0) {
+        return crc;
+    }
```

Comment:
Is this necessary even if crc2 is always 0 if len2 == 0?

Reply:
no, but I personally don't know for certain whether that always holds in Scylla or not.